docs/learn/first-dapp.mdx

```diff
@@ -0,0 +1,725 @@
+---
+title: How to build on Linea: a step-by-step tutorial to deploy your first dApp
+description: A step-by-step guide to building your first dApp on Linea.
+---
+
+## Gm developers 👋
+
+Today we're going to learn together how to create a dapp that will allow a web3 degen to stake certain tokens in your contract! 🔥
+
+We'll be using the Linea blockchain ⛓️ for this tutorial. It's a zk-rollup EVM equivalent, fully EVM-compatible and 100% proven, with advantages such as high speed and low costs.
+
+You can get some Linea Sepolia (testnet) ETH [here](https://www.infura.io/faucet/linea). 
```

Comment:
I think Matthieu's preferred flow for builders to get Sepolia ETH is to ask on Discord: https://docs.linea.build/get-started/how-to/get-testnet-eth#linea-discord